profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -251,23 +253,53 @@ enum class ProfileTabs(@StringRes val titleRes: Int) {
 }
 
 @Composable
-fun AboutTab(
+private fun AboutTab(
     viewer: Viewer,
     modifier: Modifier = Modifier
 ) {
     val scrollState = rememberScrollState()
+
+    val statsLabelToValue = listOf(
+        stringResource(R.string.total_anime) to viewer.count?.toString(),
+        stringResource(R.string.days_watched) to viewer.daysWatched?.let { "%.1f".format(it) },
+        stringResource(R.string.mean_score) to viewer.meanScore?.let { "%.1f".format(it) }
+    ).filter { it.second != null }
+
     Column(
         modifier = modifier
             .verticalScroll(scrollState)
             .padding(LocalPaddings.current.large),
         verticalArrangement = Arrangement.spacedBy(LocalPaddings.current.medium)
     ) {
-        Genres(viewer.genres)
+        StatsRow(
+            stats = statsLabelToValue,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = LocalPaddings.current.large)
+                .landscapeCutoutPadding()
+        ) {
```

Comment:
Are the items inside this content block arranged horizontally/vertically? because I don't see either `RowScope` or `ColumnScope` for it in the `StatsRow` composable.

Reply:
True but I suppose it can be either depending on `WindowClass` so I don't want to restrict it.